<?xml version="1.0"?>
<launch>

    <node pkg="hector_trajectory_server" type="hector_trajectory_server" name="trajectory_server_loam" ns="imu_pub" >
        <param name="/target_frame_name" value="map" />
        <param name="/source_frame_name" value="imu_link" />
        <param name="/trajectory_update_rate" value="10.0" />
        <param name="/trajectory_publish_rate" value="10.0" />
    </node>

    <node pkg="wheel_ins" type="pdr_node.py" name="wheel_ins" output="screen">
    </node>

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find wheel_ins)/rviz/wheel_ins.rviz" />
    </group>

    <!-- <arg name="bagfile" default="/home/huawei/Documents/dataset/rosbag/walk2.bag"/>
    <node pkg="rosbag" type="play" name="rosbag" args="$(arg bagfile) -r 2.0 -s 1.8"/> -->
</launch>